File: 657. Judge Route Circle.cpp
bool judgeCircle(char* moves) { 
    int x=0,y=0,i;
    char c;
    for(i=0;moves[i]!='\0';i++)
    {
        if(moves[i]=='R')
        {
            x++;
        }
        if(moves[i]=='L')
        {
            x--;
        }
        if(moves[i]=='U')
        {
            y++;
        }
        if(moves[i]=='D')
        {
            y--;
        }
    }
    if(x!=0||y!=0)
    {
        return false;
    }
    else
    {
        return true;
    }
}

File: Type/LeetCode_String/Easy/657. Judge Route Circle.java
class Solution {
    public boolean judgeCircle(String moves) {
        int h=0,w=0;
        for (char c : moves.toCharArray())
			switch (c) {
			case 'U':
				h++;
				break;
			case 'D':
				h--;
				break;
			case 'R':
				w++;
				break;
			case 'L':
				w--;
				break;

			}
        return h==0&&w==0;
    }
}